executor/revoke_test.go

```diff
@@ -82,7 +82,7 @@ func (s *testSuite1) TestRevokeTableScope(c *C) {
 
 	// Make sure all the table privs for new user is Y.
 	res := tk.MustQuery(`SELECT Table_priv FROM mysql.tables_priv WHERE User="testTblRevoke" and host="localhost" and db="test" and Table_name="test1"`)
-	res.Check(testkit.Rows("Select,Insert,Update,Delete,Create,Drop,Index,Alter"))
+	res.Check(testkit.Rows("Select,Insert,Update,Delete,Create,Drop,Index,Alter,Show View"))
```

Comment:
it this true in MySQL? I haven't tested it.

Reply:
From mysql-8.0.22

```
| Select,Insert,Update,Delete,Create,Drop,References,Index,Alter,Create View,Show view,Trigger |
``

Reply:
we are not supporting `trigger` now, I think it ok here.